src/amqp.rs

```diff
@@ -1,4 +1,4 @@
-//! Functionality supporting both the [sources::amqp] source and `[sinks::amqp]` sink.
+//! Functionality supporting both the `[crate::sources::amqp] source and `[crate::sinks::amqp]` sink.
```

Comment:
```suggestion
//! Functionality supporting both the `[crate::sources::amqp]` source and `[crate::sinks::amqp]` sink.
```

Missing the second backtick?

Reply:
or was this the intentionally broken one